README.md

```diff
@@ -103,6 +103,82 @@ The prl-harvester's front-end application can be developed independently of the
 
+## Running one-off harvests with a local harvester instance pointed at production Solr
+
+Certain situations may call for running (hopefully temporary) one-off harvests affecting production Solr using a local harvester instance; for example, if a repository has OAI-PMH spec compliance issues, you may want to run the harvester using a modified version of [XOAI](https://github.com/DSpace/xoai). Here's an example of such a procedure:
+
+1. Install the modified XOAI to your local Maven repository; see [here](https://github.com/UCLALibrary/xoai/tree/for-repos-that-wont-take-just-a-resumptiontoken#building-the-project) for an example.
+
+1. Update the `<harvester>` container run configuration in `pom.xml` to point to the prod Solr core:
+    ```diff
+    -<SOLR_CORE_URL>http://172.17.0.1:${test.solr.port}/solr/prl</SOLR_CORE_URL>
+    +<SOLR_CORE_URL>https://prod-solr.mylibrary.edu/solr/prl</SOLR_CORE_URL>
+    ```
+
+1. Ensure that your IP address is allowed to use the /update handler of the prod Solr core.
+
+1. Run a local instance of the harvester (and backing services, including PostgreSQL), configured with the user's local timezone and the version identifier of the modified XOAI:
+    ```bash
+    mvn clean pre-integration-test -DskipUTs -Denforcer.skip=true \
+        -Duser.timezone=America/New_York -Dxoai.version=4.2.1-SNAPSHOT
+    ```
+
+1. Log in to the local PostgreSQL instance and set the next value of the primary key of the `institutions` table to be the ID of the institution of interest on the prod instance:
+    ```bash
+    # Retrieve the ID of the institution of interest
+
+    PRL_HARVESTER_USERNAME=FIXME
+    PRL_HARVESTER_PASSWORD=FIXME
+    PRL_HARVESTER_CREDS=$(echo -n "$PRL_HARVESTER_USERNAME:$PRL_HARVESTER_PASSWORD" | base64)
+
+    INSTITUTION_ID=$( \
+        curl -s -H "Authorization: Basic $PRL_HARVESTER_CREDS" https://prl-harvester.library.ucla.edu/institutions \
+        | jq '.[] | select(.name == "University of California Los Angeles") | .id' \
+    )
+
+    # Set the next value of the primary key of the institutions table
+
+    export PGPASSWORD=$( \
+        docker container exec pgsql /usr/bin/env \
+        | grep POSTGRES_PASSWORD \
+        | sed -e s/POSTGRES_PASSWORD=//g \
+    )
+    export PGPORT=$(docker container port pgsql | grep -o -m 1 "[[:digit:]]\+$")
+
+    psql -h 0.0.0.0 -U postgres -w -c "SELECT setval(pg_get_serial_sequence('institutions', 'id'), $INSTITUTION_ID, false)"
+    ```
+
+1. Create an institution and harvest job locally:
+    ```bash
+    PRL_HARVESTER_LOCAL_INSTANCE=$(docker container port prl-harvester | cut -d ' ' -f 3 - | head -n 1)
+
+    # Make sure to provide the name, description, location, etc. as they appear on production
+    curl -X POST -H "Content-Type: application/json" "$PRL_HARVESTER_LOCAL_INSTANCE/institutions" --data-ascii '[{
+        "name": "University of California Los Angeles",
```

Comment:
Might be useful to put the full range of data for this sample institution to indicate what sort of data needs to be supplied.

Reply:
✔️ thanks for the feedback.